scalafix/rules/src/main/scala/rsc/rules/semantics/Scopes.scala

```diff
@@ -25,6 +25,34 @@ sealed trait Scope {
   }
 }
 
+final class SyntheticScope extends Scope {
+
+  def lookup(name: n.Name): String = additionalNames.getOrElse(name, "")
+
+  def addImport(sym: String): Unit = {
+    if (!additionalImports.contains(sym)) {
+      getImporter(sym) foreach { importer =>
+        val name = sym.desc.name
+        additionalNames += (name -> sym)
+        additionalImports += (sym -> importer)
+      }
+    }
+  }
+
+  def imports: Seq[Importer] = additionalImports.values.toSeq
+
+  private var additionalNames = mutable.Map.empty[n.Name, String]
+
+  private var additionalImports = mutable.Map.empty[String, Importer]
+
+  private def getImporter(sym: String): Option[Importer] = {
```

Comment:
`getImporter` is too generic of a name. How about simply inlining it into its single callsite?